plugins/sqlmap.rb

```diff
@@ -28,30 +28,29 @@ def commands
 
       def cmd_sqlmap_connect(*args)
         if args.length == 0
-          print_error('Need a host, and optionally a port')
+          print_error("Need a host, and optionally a port")
```

Comment:
The only reason I used double quotes was for consistency. A lot of times print_* methods use string interpolation and if single quotes are used, the variables are not highlighted. The above calls don't have any string interpolation but I thought I should go with double quotes in every print_* call. Will change the quotes as per ruby style guide. Thanks